cdap-security-spi/src/test/java/co/cask/cdap/security/spi/authorization/AuthorizerTest.java

```diff
@@ -46,6 +49,10 @@ public void testSimple() throws UnauthorizedException {
     authorizer.grant(namespace, user, Collections.singleton(Action.READ));
     authorizer.enforce(namespace, user, Action.READ);
 
+    Set<Privilege> expectedPrivileges = new HashSet<>();
```

Comment:
Yes its because we are not supporting RBAC in DatasetBasedAuthorizer. I have a jira which tracks if we want to support it or not and if we do then implement it. When we implement that we will be adding tests here. We cannot add tests here right now.